src/ci/docker/host-x86_64/x86_64-gnu-tools/checktools.sh

```diff
@@ -17,6 +17,7 @@ python3 "$X_PY" test --stage 2 --no-fail-fast \
     src/tools/rls \
     src/tools/rustfmt \
     src/tools/miri \
+    src/librustdoc
```

Comment:
This should also check for changes to the testsuite (like the ones in this PR).
```suggestion
    src/librustdoc src/rustdoc-json-types src/test/rustdoc{,-ui,-js,-gui,-json,-js-std}
```
(and you'll also need to change the shebang to #!/bin/bash to support globbing)

Reply:
Oh wait hold on, this isn't the right thing to modify - this is an `x.py test` command. You want to update the logic that causes this to run in the first place - there's a comment about TOOLSTATE_FILE, maybe search for that.

Reply:
I think it refers to `src/bootstrap/toolstate.rs` which has `STABLE_TOOLS` and `NIGHTLY_TOOLS` arrays that contains lists somewhat similar. However, I don't see how it relates to this... And I'm not even sure how this helps in our case...